packages/nixos/debug.nix

```diff
@@ -32,7 +32,9 @@ in
 
     services.getty.autologinUser = "root";
 
-    boot.kernelParams = [ "console=ttyS0" ];
+    # required for local booting, but no boot logs in kata with this
+    boot.kernelParams = lib.optionals config.contrast.azure.enable [ "console=ttyS0" ];
```

Comment:
I don't understand: the serial console worked fine on BM, why do we disable it?

Reply:
QEMU, locally, without specifying a plethora of `virtconsole` options, will write to `ttyS0`. Kata reads the guest logs from `hvc*`, which doesn't work anymore when specifying `ttyS0` here. (Even though I think it should, but maybe more than 2 consoles are not supported by the kernel.)